src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -180,6 +194,16 @@ class GazeDataFrame:
     └──────┴────────────┘
     """
 
+    frame: pl.DataFrame
+
+    events: pm.EventDataFrame
```

Comment:
`pm` is needed due to circular import?

Reply:
yes unfortunately, will take a while until I manage to resolve this